core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionActor.scala

```diff
@@ -290,6 +294,13 @@ trait WorkflowSubmission extends FutureSupport with LazyLogging with MethodWiths
         wdl <- getWdl(methodConfig, userCredentials)
       } yield {
         val wfOpts = buildWorkflowOpts(workspaceRec, submissionRec.id, RawlsUser(RawlsUserSubjectId(userIdInfo), RawlsUserEmail(submissionRec.submitterEmail)), petSAJson, billingProject, submissionRec.useCallCache, WorkflowFailureModes.withNameOpt(submissionRec.workflowFailureMode))
+        val submissionAndWorkspaceLabels = Map("submission-id" -> submissionRec.id.toString,  "workspace-id" -> workspaceRec.id.toString)
+        val wfLabels = workspaceRec.workflowCollection match {
+          case None => submissionAndWorkspaceLabels
+          case Some(workflowCollection) => if (useWorkflowCollectionField) submissionAndWorkspaceLabels else submissionAndWorkspaceLabels + ("caas-collection-name" -> workflowCollection)
+        }
+        val wfCollection = if (useWorkflowCollectionField) workspaceRec.workflowCollection else None
+
```

Comment:
I chose `submission-id` and `workspace-id` for the labels and the ticket seemed to specify `caas-collection-name` for the workflow collection label name. I'm unsure why is specifies "caas"?

Reply:
cromwell as a service. the cromfolks made that up

Reply:
the label is read and interpreted by cromiam, so you have to call it what they're looking for